docs/applications.md

```diff
@@ -246,11 +246,11 @@ your pusher.
 
 #### Notification format
 
-The notification as received by your web application will contain these keys 
-if they were set by the homeserver, and otherwise omit them. These are the
-same as specified in [the push gateway spec]
-(https://matrix.org/docs/spec/push_gateway/r0.1.0#post-matrix-push-v1-notify),
-but `counts` with `unread` and `missed_calls` is flattened into the notification object.
+The notification as received by your web application will contain the following keys 
+(assuming they were sent by the homeserver). These are the
```

Comment:
I meant more that if they are null in the notification from the homeserver, they will be omitted in the payload to the push gateway. Not sure the distinction is important though.

Reply:
Ah, I misunderstood this then. I don't know if that distinction is important, yeah. I think the important bit is that you should handle them not being there?